javalin/src/main/java/io/javalin/http/staticfiles/StaticFileConfig.java

```diff
@@ -0,0 +1,19 @@
+package io.javalin.http.staticfiles;
+
+import io.javalin.core.util.Header;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.eclipse.jetty.server.handler.ContextHandler;
+import org.jetbrains.annotations.NotNull;
+import static java.util.stream.Collectors.toMap;
+
+public class StaticFileConfig {
+    // @formatter:off
+    public @NotNull String hostedPath = "/";
+    public @NotNull String directory = "/public";
+    public @NotNull Location location = Location.CLASSPATH;
+    public boolean precompress = false;
+    public ContextHandler.AliasCheck aliasCheck = null;
+    public Map<String, String> headers = Stream.of("").collect(toMap(x -> Header.CACHE_CONTROL, x -> "max-age=0")); // mapOf(Header.CACHE_CONTROL to "max-age=0")
```

Comment:
It looks kinda tricky. For readability and further extension I'd use:

```java
public Map<String, String> headers = new HashMap<String, String>() {{
  put(key, value);
}}
```

or for entry just

```java
public Map<String, String> headers = new HashMap<>(Collections.singletonMap(key, value));
```

Reply:
Thanks, I didn't know about `Collections.singletonMap(key, value)` !